packages/reports/addon/models/bard-request/request.js

```diff
@@ -459,6 +459,27 @@ export default Fragment.extend(Validations, {
     return this.removeSort(sortObj);
   },
 
+  /**
+   * Removes the sort from the sort array using the metric model and parameters
+   *
+   * @method removeSortMetricByModelAndParam
+   * @param {DS.Model} metricModel
+   * @param {Object} [parameters] - parameters [optional]
+   * @returns {Void}
+   */
+  removeSortMetricByModelAndParam(metricModel, parameters) {
+    if (parameters) {
+      let canonicalizedMetric = canonicalizeMetric({
```

Comment:
I think we can just call `get(metricModel, 'canonicalName')`  instead of `canonicalizeMetric`

Reply:
so actually we don't have `canonicalName` here, maybe that's why `removeRequestMetricWithParam` was written the same way.